packages/web/src/utils.ts

```diff
@@ -116,3 +116,58 @@ function turnValueToRegExp(value: string): RegExp {
 function escapeRegExp(string: string): string {
   return string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
 }
+
+export function getBasePath(): null | string {
+  // !! important !!
+  // do not access env variables using process.env[varname] or import.meta.env[varname].
+  // some bundles won't replace the value at build time.
+
+  // vite-powered apps (sveltekit, nuxt, vue, astro, remix)
+  try {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment, @typescript-eslint/prefer-ts-expect-error -- only TS during build time will complain.
+    // @ts-ignore -- Rollup will fail to generate d.ts because it doesn't know import.meta.env.
+    const viteValue = import.meta.env
+      .VITE_VERCEL_OBSERVABILITY_BASEPATH as string;
```

Comment:
No we don't have to, because `import` doesn't behave like `process`. `process` is a variable that belongs to the global namespace, while `import` is a keyword. 
So it doesn't have a type, that's why I've wrapped it within a try-catch.